### DISCUSSION TITLE
by default CI of the forecast plots

### DISCUSSION THREAD

**Discussion by SamuelBeauregard** [nicholasjclark/mvgam#65] [Q&A] [answered]
Silly question, but I can't find the answer anywhere. What are the by default CI (the different shades of red on the plot) for the forecast plots? Is it 2.5, 25, 50, 75 and 97.5% ?

![image](https://github.com/user-attachments/assets/8177c8fa-3ca7-4f72-b02c-b945000175cd)

Thank you once again!

**Comment by nicholasjclark** [comment, marked as answer]
They are 90, 60, 40, 20, median: `probs = c(0.05, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.95)`. Eventually I'll move all plots to ggplot so these can be controlled by the user

**Reply by SamuelBeauregard** [reply]
This feels like an extremely basic question, but just to make sure I get this right, the 5th point is on the median in the 90% CI?

**Reply by nicholasjclark** [reply]
Yes correct. The 4 ribbons show 90, 60, 40 and 20th posterior intervals while the line shows the posterior median. You can always extract the full posterior by using the forecast() function and pulling out the relevant matrix from the returned object. This would allow you to calculate any intervals you need
